keycloakapi/src/main/kotlin/de/klg71/keycloakmigration/keycloakapi/KeycloakClient.kt

```diff
@@ -1,46 +1,6 @@
 package de.klg71.keycloakmigration.keycloakapi
 
-import de.klg71.keycloakmigration.keycloakapi.model.AddClient
-import de.klg71.keycloakmigration.keycloakapi.model.AddClientScope
-import de.klg71.keycloakmigration.keycloakapi.model.AddFlow
-import de.klg71.keycloakmigration.keycloakapi.model.AddFlowExecution
-import de.klg71.keycloakmigration.keycloakapi.model.AddGroup
-import de.klg71.keycloakmigration.keycloakapi.model.AddIdentityProvider
-import de.klg71.keycloakmigration.keycloakapi.model.AddMapper
-import de.klg71.keycloakmigration.keycloakapi.model.AddRealm
-import de.klg71.keycloakmigration.keycloakapi.model.AddRole
-import de.klg71.keycloakmigration.keycloakapi.model.AddSimpleClient
-import de.klg71.keycloakmigration.keycloakapi.model.AddUser
-import de.klg71.keycloakmigration.keycloakapi.model.AddUserFederation
-import de.klg71.keycloakmigration.keycloakapi.model.AddUserFederationMapper
-import de.klg71.keycloakmigration.keycloakapi.model.AssignClientScope
-import de.klg71.keycloakmigration.keycloakapi.model.AssignGroup
-import de.klg71.keycloakmigration.keycloakapi.model.AssignRole
-import de.klg71.keycloakmigration.keycloakapi.model.AuthenticationExecution
-import de.klg71.keycloakmigration.keycloakapi.model.AuthenticatorConfig
-import de.klg71.keycloakmigration.keycloakapi.model.Client
-import de.klg71.keycloakmigration.keycloakapi.model.ClientListItem
-import de.klg71.keycloakmigration.keycloakapi.model.ClientScope
-import de.klg71.keycloakmigration.keycloakapi.model.ClientScopeItem
-import de.klg71.keycloakmigration.keycloakapi.model.ClientSecret
-import de.klg71.keycloakmigration.keycloakapi.model.Flow
-import de.klg71.keycloakmigration.keycloakapi.model.Group
-import de.klg71.keycloakmigration.keycloakapi.model.GroupListItem
-import de.klg71.keycloakmigration.keycloakapi.model.IdentityProvider
-import de.klg71.keycloakmigration.keycloakapi.model.ImportClientRepresentation
-import de.klg71.keycloakmigration.keycloakapi.model.Mapper
-import de.klg71.keycloakmigration.keycloakapi.model.Realm
-import de.klg71.keycloakmigration.keycloakapi.model.RegisterRequiredActionProvider
-import de.klg71.keycloakmigration.keycloakapi.model.RequiredActionProviderItem
-import de.klg71.keycloakmigration.keycloakapi.model.Role
-import de.klg71.keycloakmigration.keycloakapi.model.RoleListItem
-import de.klg71.keycloakmigration.keycloakapi.model.UpdateFlow
-import de.klg71.keycloakmigration.keycloakapi.model.UpdateFlowExecution
-import de.klg71.keycloakmigration.keycloakapi.model.UpdateGroup
-import de.klg71.keycloakmigration.keycloakapi.model.User
-import de.klg71.keycloakmigration.keycloakapi.model.UserFederation
-import de.klg71.keycloakmigration.keycloakapi.model.UserFederationMapper
-import de.klg71.keycloakmigration.keycloakapi.model.UserGroupListItem
+import de.klg71.keycloakmigration.keycloakapi.model.*
```

Comment:
Star-Imports should be forbidden by the build process but I will check that again

Reply:
The build complained about a star import in ActionFactory but not in the api.

Reply:
done